scss/component/_item.scss

```diff
@@ -1,39 +1,70 @@
 // Item Styles
 // ===========
 
-[data-sassdoc-item] {
+.item {
   border-top: 2px dotted color('border');
-  margin: size('gutter') 0;
+  margin: size('shim') 0;
   padding: size('gutter') 0;
 }
 
-[data-sassdoc='item-type'],
-[data-sassdoc='var-type'] {
-  color: color('light');
-  font-weight: normal;
+[data-item-section] {
+  margin-bottom: size('double-gutter');
 }
 
-[data-sassdoc='item-description'] {
-  margin-bottom: size('gutter');
+[data-item-section='header'] {
+  margin: size('gutter') 0;
 }
 
-[data-sassdoc-display] {
-  margin: size('gutter') 0;
+.item-title {
+  font-size: size('h3');
 }
 
-[data-sassdoc-table] {
+.item-subtitle {
+  border-bottom: 1px dotted color('light-gray');
+  color: color('text-light');
   font-size: size('small');
+  font-weight: normal;
+  letter-spacing: 0.05em;
+  margin-bottom: size('shim');
+  text-transform: uppercase;
+}
 
-  td,
-  th {
-    padding: size('half-shim') size('shim');
+.item-type,
+.item-name,
+.item-value,
+.alias-title {
+  @include font-family('code');
+  display: inline-block;
+}
 
-    @include below('page') {
-      padding-left: 0;
-    }
-  }
+.item-type,
+.item-value,
+.value-type,
+.item-note {
+  color: color('text-light');
+  display: inline-block;
+  font-weight: normal;
+}
+
+.alias {
+  color: color('text-light');
+  font-style: italic;
+}
+
+.alias-title {
+  font-style: normal;
+  font-weight: bold;
+}
+
+.param-list {
+  margin-bottom: size('shim');
+}
+
+.param-title {
```

Comment:
This feels a little tight to me. Could we add a half-shim margin-bottom or maybe it would be better to add a margin-top to the `.param-details` instead?